scripts/localStorageUtils.js

```diff
@@ -1,5 +1,5 @@
 const getLocalStorage = (key) => {
-  const data = window.localStorage.getItem(key);
+  const data = window.sessionStorage.getItem(key);
```

Comment:
Function name - `getLocalStorage` and usage of `sessionStorage` within function body doesn't match up

Reply:
Note that changing from 'localStorage' to 'sessionStorage` will affect the implementation of `review` component since this file is being used there as well. Please confirm the behaviour change with Product team as well